boards/riscv/beagle_v_starlight/doc/index.rst

```diff
@@ -0,0 +1,75 @@
+.. _beagle_v_starlight:
```

Comment:
as per @pdp7's recommendation, I would change to `beaglev_starlight_jh7100`

Reply:
Fixed.